tests/test_main.py

```diff
@@ -3,16 +3,90 @@
 import pyglet
 import pytest
 
-from tempus_fugit_minecraft.main import Window
+from tempus_fugit_minecraft.main import Window, Model, LIGHT_CLOUD
 
 
 @pytest.fixture(scope="class")
 def window():
-    window = Window()
-    return window
+    yield Window()
+
+
+@pytest.fixture(scope="class")
+def model():
+    yield Model()
```

Comment:
this actually very helpful as we won't bother creating instances of Window and Model classes on each method.
Good job on this.